core/arch/arm/mm/tee_pager.c

```diff
@@ -233,6 +233,9 @@ void *tee_pager_phys_to_virt(paddr_t pa)
 	vaddr_t v;
 	size_t n;
 
+	if (pa & CORE_MMU_PGDIR_MASK || len > CORE_MMU_PGDIR_SIZE)
```

Comment:
Should be `SMALL_PAGE_MASK` and `SMALL_PAGE_SIZE`, no?

Reply:
Yes, now PGDIR looks strange for me too)